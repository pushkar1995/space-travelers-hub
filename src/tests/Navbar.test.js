import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Header from '../components/Navbar/Navbar';

describe('Header Component', () => {
  it('displays the correct navigation links', () => {
    render(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
    );

    const rocketsLink = screen.getByText('Rockets');
    const missionsLink = screen.getByText('Missions');
    const profileLink = screen.getByText('Profile');

    expect(rocketsLink).toMatchSnapshot();
    expect(missionsLink).toMatchSnapshot();
    expect(profileLink).toMatchSnapshot();
  });
});
