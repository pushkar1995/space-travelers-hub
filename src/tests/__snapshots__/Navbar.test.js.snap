// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component displays the correct navigation links 1`] = `
<a
  class="active"
  href="/"
>
   
  Rockets
   
</a>
`;

exports[`Header Component displays the correct navigation links 2`] = `
<a
  class=""
  href="/missions"
>
   
  Missions
   
</a>
`;

exports[`Header Component displays the correct navigation links 3`] = `
<a
  class=""
  href="/myprofile"
>
   
  Profile
   
</a>
`;
